addons/binding/org.openhab.binding.pulseaudio/src/main/java/org/openhab/binding/pulseaudio/internal/cli/Parser.java

```diff
@@ -35,8 +35,7 @@
     private static final Logger logger = LoggerFactory.getLogger(Parser.class);
 
     private static final Pattern pattern = Pattern.compile("^\\s+([a-z\\s._]+)[:=]\\s*<?\"?([^>\"]+)\"?>?$");
-    private static final Pattern volumePattern = Pattern.compile(
-            "^(0|front-left):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?(1|front-right):( *[0-9]+ \\/)? *([0-9]+)% *\\/? *([0-9\\-, dB]+)?$");
+    private static final Pattern volumePattern = Pattern.compile("^([\\w\\-]+):( *[\\d]+ \\/)? *([\\d]+)% *\\/? *([\\d\\-., dB]+)?$");
```

Comment:
Understood.  This change should actually be more permissive than just adding . would be though, and my specific use case does involve a 7.1 surround card (although I can live with support for just stereo channels...).  Is there any specific reason why the cli interface was chosen, instead of the native tcp interface?  It seems to me the cli interface is almost certainly going to get easily broken by upstream.